src/moepkg/exmode.nim

```diff
@@ -237,6 +237,13 @@ proc isForceQuitCommand(command: seq[seq[Rune]]): bool {.inline.} =
 proc isShellCommand(command: seq[seq[Rune]]): bool {.inline.} =
   return command.len >= 1 and command[0][0] == ru'!'
 
+proc isBackgroundCommand(command: seq[seq[Rune]]): bool {.inline.} =
+  return command.len == 1 and cmpIgnoreCase($command[0], "bg") == 0
+
+proc isManualCommand(command: seq[seq[Rune]]): bool {.inline.} =
+  # TODO:  Configure a default manual page to show on `:man`.
+  return command.len > 1 and cmpIgnoreCase($command[0], "man") == 0
```

Comment:
@fox0430

`command.len >= 1` allows for `:man` and thus the configuration of a default manual page to show.  I would like to suggest to either redirect to `man`'s manual page or Moe's one (as soon as it was written).

This is opted out for now but can be changed very easily.